django/pyrog/api/serializers.py

```diff
@@ -3,15 +3,19 @@
 from pyrog import models
 
 
-class SourceSerializer(serializers.ModelSerializer):
+class ResourceSerializer(serializers.ModelSerializer):
```

Comment:
You can just add `"resources"` to the included fields. Less code, same result

```suggestion
class SourceSerializer(serializers.ModelSerializer):
    class Meta:
        model = models.Source
        fields = ["id", "name", "version", "updated_at", "created_at", "resources"]


class ResourceSerializer(serializers.ModelSerializer):
    class Meta:
        model = models.Resource
        fields = "__all__"


```